schema/mutation.js

```diff
@@ -4,7 +4,32 @@ const Web3 = require("web3");
 const Tx = require("ethereumjs-tx");
 const rpcURL = process.env.INFURA;
 const web3 = new Web3(rpcURL);
+const transaxFunc = (data, callback)=>{  web3.eth.getTransactionCount(account1, (err, txCount) => {
+    // Build the transaction
```

Comment:
Fine for now, but let's remove console.log's before we push to production from development

Reply:
Done!